hadoop-hdds/dev-support/checkstyle/checkstyle.xml

```diff
@@ -154,18 +140,15 @@
         <module name="ParenPad"/>
         <module name="TypecastParenPad"/>
         <module name="WhitespaceAfter">
-          <property name="tokens" value="COMMA, SEMI"/>
+            <property name="tokens" value="COMMA, SEMI"/>
         </module>
         <module name="WhitespaceAround"/>
 
-
         <!-- Modifier Checks                                    -->
         <!-- See http://checkstyle.sf.net/config_modifiers.html -->
         <!-- This one is nitty, disable -->
-        <!-- <module name="ModifierOrder"/> -->
         <module name="RedundantModifier"/>
```

Comment:
Looks like the comment `This one is nitty, disable` applies to the rule being removed, so it should be also removed.